week-5/C-promises/exercise.js

```diff
@@ -55,7 +62,16 @@ function exercise3() {
   YOUR NAME"
 */
 function exercise4() {
-  var promise4 = concatPromise()
+  var promise4 = concatPromise();
+  promise4
+    .then(firstResolvedValue => {
+      return new Promise(resolve => {
```

Comment:
There is no need to create a new Promise, it happens automatically. You can just do 
```suggestion
    promise4.then(firstResolvedValue => firstResolvedValue + "Saeen")
```

Reply:
I did that because in the question is asked to return the concatenated string as a promise and do the rest on it.

Reply:
You can return the concatenated string, it gets wrapped in a Promise automatically. I will have to re-read the question but I don't think it wanted you to explicitly crate the Promise since there is no need.